### ISSUE TITLE
Store name and house number only once for clipped polygons

### ISSUE THREAD

**Issue by mapsforge** [mapsforge/mapsforge#263]
_From [thilo.mu...@gmail.com](https://code.google.com/u/109067824594513388759/) on March 04, 2012 17:31:09_

If a polygon has been clipped to more than one base tile, the name and the house number of the original way should only be stored for one of the polygons, not for each. This has two advantages:
1) the size of the resulting file will be smaller
2) the renderer will not draw the same label several times close to each other

To decide on which of the clipped polygons the strings should be stored, the centroid position of the original polygon should be used.

_Original issue: http://code.google.com/p/mapsforge/issues/detail?id=263_

**Comment by devemux86** [comment]
Mapsforge has changed a lot since then, can create new issue if still valid.
